src/Hl7.Fhir.Specification/Validation/XmlValidationExtensions.cs

```diff
@@ -6,7 +6,9 @@
  * available at https://raw.githubusercontent.com/FirelyTeam/firely-net-sdk/master/LICENSE
  */
 
-#nullable enable
+#nullable enable annotations
```

Comment:
Changing line 100 to
```
instance.Validate(xsdSchemas, (o, args) => { result.AddIssue(ToIssueComponent(args)); });
```

will remove the nullable warning and we don't have to disable the nullable warnings.
To be honest: I don't know why this solves the warning, but I followed the example [here](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmldocument.validate?view=net-5.0#system-xml-xmldocument-validate(system-xml-schema-validationeventhandler)).